docs/root/intro/arch_overview/observability/access_logging.rst

```diff
@@ -11,6 +11,31 @@ features:
 * Customizable access log filters that allow different types of requests and responses to be written
   to different access logs.
 
+.. _arch_overview_access_log_filters:
+
+Access log filters
+==================
+
+Envoy supports several built-in access log filters. Currently supported filters are:
```

Comment:
Can you ref link into the actual filter config docs? It's up to you if you want to replicate all of the filters here as they may get out of date vs. just link to the filter structure. I mostly just wanted to call out that extension filters are available. https://www.envoyproxy.io/docs/envoy/latest/api-v2/config/filter/accesslog/v2/accesslog.proto#envoy-api-msg-config-filter-accesslog-v2-accesslog

Reply:
I'll be happy to ref link into the actual filter config docs, but I can't figure out how. 
Is there an example that I could follow?

Reply:
@efimki see how this is done here: https://github.com/envoyproxy/envoy/blame/master/docs/root/configuration/runtime.rst#L22

Reply:
Thanks! I think I've done the right thing now, but not 100% sure. 

I'm totally confused about msg vs field in references like <envoy_api_msg_config.filter.accesslog.v2.AccessLogFilter> <envoy_api_field_config.filter.accesslog.v2.AccessLogFilter.extension_filter>.